addons/mail/static/tests/qunit_suite_tests/components/composer_suggestion_partner_tests.js

```diff
@@ -13,23 +13,22 @@ QUnit.module('composer_suggestion_partner_tests.js', {
 QUnit.test('partner mention suggestion displayed', async function (assert) {
     assert.expect(1);
 
-    this.data['mail.channel'].records.push({ id: 20 });
-    const { createComposerSuggestionComponent, messaging } = await start({ data: this.data });
-    const thread = messaging.models['Thread'].findFromIdentifyingData({
-        id: 20,
-        model: 'mail.channel',
+    const pyEnv = await startServer();
+    const channelId = pyEnv['mail.channel'].create({});
+    const { afterNextRender, openDiscuss } = await start({
+        discuss: {
+            params: {
+                default_active_id: channelId,
+            },
+        },
+        hasDiscuss: true,
     });
-    const partner = messaging.models['Partner'].create({
-        id: 7,
-        im_status: 'online',
-        name: "Demo User",
-    });
-    await createComposerSuggestionComponent(thread.composer, {
-        isActive: true,
-        modelName: 'Partner',
-        recordLocalId: partner.localId,
+    await openDiscuss();
+    await afterNextRender(() => {
+        document.querySelector('.o_ComposerTextInput_textarea').focus();
+        document.execCommand('insertText', false, "@");
     });
-
+    // Should show mitchell admin
```

Comment:
maybe use `@mitchel` above then, relying on the number of existing partners is a bit too weak

Should do .length > 0 also if the number if irrelevant, like you did in another test